string name
vision_msgs/BoundingBox3D bb3d
